test/units/plugins/filter/test_mathstuff.py

```diff
@@ -78,9 +78,10 @@ def test_max(self):
 
 class TestLogarithm:
     def test_log_non_number(self):
-        with pytest.raises(AnsibleFilterError, message='log() can only be used on numbers: a float is required'):
+        # Message changed in python3.6
+        with pytest.raises(AnsibleFilterError, match='log\\(\\) can only be used on numbers: (a float is required|must be real number, not str)'):
```

Comment:
@abadger things like these would read better with a raw-string literal (which is common for regexes):
```suggestion
        with pytest.raises(
        	AnsibleFilterError,
        	match=r'log\(\) can only be used on numbers: (a float is required|must be real number, not str)',
        ):
```